### PR TITLE
WebP + CocoaPods + use_frameworks! Redux

### PR DESCRIPTION
**Pull request by Adlai-Holler**
This supersedes #106. Tests pass locally and also I was able to use WebP in a Swift project with CocoaPods + use_frameworks!

### CODE REVIEW

File: Example/Podfile [hunk 1]
```diff
@@ -1,3 +1,5 @@
+xcodeproj 'PINRemoteImage.xcodeproj'
```
Comment (Adlai-Holler) [inline Example/Podfile]: I initially built this with CocoaPods 0.39.0 and that version required this line. The current version for the project, 0.38.2, doesn't require it but also doesn't mind it so I left it. I can remove if needed.

**Comment by Adlai-Holler** [comment]
Hate to intermingle pod sandbox updates with code changes like this, but it's the only way to get the tests running. The meaningful changes are all in d33db52.

**Comment by Adlai-Holler** [comment]
OK so the remaining test failure is, on the iOS 8.4 simulator `testQOS` just timed out. I don't think it's related to this PR since the test didn't time out on 9.2 and I don't think I affected any of its tested code.

**Comment by garrettmoon** [comment]
@Adlai-Holler the Travis CI builds seem to time out rather frequently, I restarted the failed one.

**Comment by garrettmoon** [comment]
Thank you for doing this! I had to fix this up a bit for Mac support and the examples.

**Comment by garrettmoon** [comment]
@Adlai-Holler I don't know why this says it's not merged, please confirm it is and then close.

**Comment by Adlai-Holler** [comment]
Yep, looking good. Love this framework btw.

**Comment by garrettmoon** [comment]
@Adlai-Holler Thank you for helping make it even better :)

**Pull request closed without merging**